app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_counter
+
+  private
+
+  def update_counter
+    author.increment!(:likes_counter)
```

Comment:
- [ ] Same thing as in the post model, here it would be better to use `post.increment` instead of `author.increment` so please work on this issue.